code/AEPSDK.brs

```diff
@@ -247,6 +247,48 @@ function AdobeAEPSDKInit(taskNode = invalid as dynamic) as object
             m._private.dispatchEvent(event)
         end function,
 
+        ' *************************************************************************************
+        '
+        ' set consent.
+        '
+        ' This function will send the consent data in XDM format following Adobe 2.0 standard to the Edge network.
+        '
+        '
+        ' @param data as object : containing consent XDM data following Adobe 2.0 standard
+        ' @param [optional] callback as function(context, result) : handle Edge consent response
+        ' @param [optional] context as dynamic : context to be passed to the callback function
+        '
+        ' *************************************************************************************
+
+        setConsent: function(data as object, callback = _adb_defaultCallback as function, context = invalid as dynamic) as void
```

Comment:
Is there a specific reason why we accept a callback for the setConsent API, given that we do not accept one for the Mobile SDK? Are there scenarios where the application parses the response and responds accordingly?

Reply:
I initially added it to match with sendEvent API but yes as per our discussion I don't see much use case for callbacks here and there are no callbacks for setConsent on Alloy SDK or update consent on Mobile SDK. 

@kevinlind Do you have any thoughts on this? If we all agree I can go ahead and update the API and remove the callback.